### PR TITLE
readme: added license badge

### PR DESCRIPTION
**Pull request by ipa-hsd**
Related to https://github.com/ros-industrial/ros_industrial_issues/issues/57

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```
Comment (miguelprada) [inline README.rst]: Readme uses rst, so this isn't rendering properly.

Probably something like:

```
.. image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
    :target: https://opensource.org/licenses/Apache-2.0
    :alt: License
```

Comment (miguelprada) [reply README.rst]: Fixed in e139e04

**Comment by gavanderhoorn** [comment]
[rendered file](https://github.com/ros-industrial/industrial_ci/blob/e139e04205e07dfdf98599f612ab3fa7e1a4cef7/README.rst).

**Comment by ipa-mdl** [comment]
@gavanderhoorn:

> rendered file.

Still wrong, the quickstart entries are gone (which is unrelated to this change)

> should the badge be under the main header?

No, I think it's fine.

Just holding the merge back to not clog the Travis CI build queue too much ^^

**Comment by gavanderhoorn** [comment]
All other repositories have the license badge(s) under the initial heading. Compare [fanuc](https://github.com/ros-industrial/fanuc/blob/indigo-devel/readme.md) fi.

**Comment by miguelprada** [comment]
Moved the badge under the initial heading since it has the side effect of [restoring the preview](https://github.com/ipa-hsd/industrial_ci/tree/2472c759b186086f82a7efd1b370fc6962b2dcb4). Let me know if you'd like me to revert.

**Pull request merged**